src/Models/Context/DeeplinkContext/DeeplinkContext.tsx

```diff
@@ -197,3 +216,44 @@ const parseArrayParam = (value: string): string[] => {
     if (!value) return undefined;
     return value.split(ARRAY_VALUE_SEPARATOR);
 };
+
+// START of local storage handling
+/**
+ * read the selected environment url from local storage if exists to set the initial value of 'adtUrl' in the initial default state of DeeplinkContext
+ */
+const getSelectedEnvironmentUrlFromLocalStorage = () => {
+    try {
+        return (JSON.parse(
```

Comment:
can we store these 2 data points in a single object in the storage? it seems like they can be persisted together as part of the app state. 
`environment:{adtUrl:string, storageUrl:string}`

Reply:
when we say "environment" it usually refers to the ADT environment (which is an ADT "instance" in technical terms) - at least this was what we were referring to. If we are going to do this change it requires other changes for container url in the local storage. Also, the reason why we kept the adt url in a certain way in the localstorage was to be consistent with the ADT explorer app (they use {appAdtUrl: <selected_adt_instance_url>} in localstorage). 

What do you think @darsney about changing the structure of localstorage for selected url information?